### DISCUSSION TITLE
problema con tamaño de las piezas a cortar

### DISCUSSION THREAD

**Discussion by joseval45** [arkypita/LaserGRBL#1794] [General]
Diubjo con el programa rhinocero convirtiendo los archivos en svg.  la cortadora laser es la sculpfun s9 . no tngo problemas con piezas pequeñas pero pasando de 30 cm el programa laser grbl no me reconoce los archivos, en teoria la cortadora tiene capacidad de corte en 410mm por 450 mm. supongo que es problema de configuracion alguien me puede ayudar. gracias noy novato en esto.

**Comment by Tribo-2dot0** [comment]
Check these two values:
![Captura de Tela (38)](https://user-images.githubusercontent.com/83982525/158614723-5d5d0ab4-b373-46fe-96fd-3f9128de8dbf.png)

**Reply by joseval45** [reply]
los valores que tengo en esas lineas son 410 y 420 que es el area de trabajo de la sculpfun s9 .  No he comentado antes que el problema lo tengo desde hace dos dias antes no he tenido problemas con los archivos grandes de hecho he llegado a cortar sin problemas, los antiguas me los abre y trabaa pero los que creo ahora cuando intento abrirlos no aparecen .si activo el puntero aparece en las cordenadas 0,0 y si activo el mostrar area trabajo se vuelve a encender brevemente el puntero como si el archivo fuese minusculo-
